package.json

```diff
@@ -13,7 +13,7 @@
     "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
     "watch-sass": "npm run compile-sass -- --watch",
     "build": "npm run compile-sass && tsc && npm run copy-views",
-    "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
+    "start": "node -r dotenv/config dist/server.js | bunyan -o short",
     "start:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
```

Comment:
A bit worried that $NODE_OPTIONS is removed. Can it keep both $NODE_OPTIONS and `-r dotenv/config` ?

Reply:
good point - have added it back